torchreid/engine/image/am_softmax.py

```diff
@@ -504,9 +504,9 @@ def find_lr(
                 optimizer = self.optims[name]
             if epochs_warmup != 0:
                 state_cacher = StateCacher(in_memory=True, cache_dir=None)
-                state_cacher.store("model", self.models[name].module.cpu().state_dict())
+                state_cacher.store("model", get_model_attr(self.models[name], 'cpu')().state_dict())
                 state_cacher.store("optimizer", self.optims[name].state_dict())
-                self.models[name].module.to(model_device)
+                get_model_attr(self.models[name], 'to')(model_device)
```

Comment:
Did you check it runs on CPU?

Reply:
Tests are passed in OTE for all mobilenets. So, it was verified on CPU.